parlai/agents/drqa/utils.py

```diff
@@ -7,6 +7,20 @@
 import time
 import unicodedata
 from collections import Counter
+import spacy
```

Comment:
Since we already load a spacy model as part of the DrQA SimpleDictionary, I think we should just have these as an extra function -- e.g. `features(pos=False, ner=False)`. And those fields should be passed in as part of the `ex` dict to vectorize.